include/envoy/network/listener.h

```diff
@@ -31,6 +32,12 @@ class ListenerConfig {
    */
   virtual FilterChainFactory& filterChainFactory() PURE;
 
+  /**
+   * @return QuicListenerFactory* the factory for creating QUIC listeners, or nullptr if none is
+   *         configured.
+   */
+  virtual Quic::QuicListenerFactory* quicListenerFactory() PURE;
```

Comment:
What is the ownership model for this pointer?

Reply:
It's owned internally.